install/kubernetes/app.yaml

```diff
@@ -11,17 +11,17 @@ spec:
     matchLabels:
       app: backstage
       component: frontend
-    template:
-      metadata:
-        labels:
-          app: backstage
-          component: frontend
-      spec:
-        containers:
-        - name: app
-          image: spotify/backstage:latest
-          imagePullPolicy: Always
-          ports:
-          - containerPort: 80
-            name: app
-            protocol: TCP
+  template:
+    metadata:
+      labels:
+        app: backstage
+        component: frontend
+    spec:
+      containers:
+      - name: app
+        image: spotify/backstage:latest
+        imagePullPolicy: IfNotPresent
```

Comment:
allows you to pull local image. But I don't think spotify publishes a public image for backstage. All I see is other backstage images from various users which appear out of date.